src/Response.php

```diff
@@ -149,7 +159,9 @@ class Response extends AbstractMessage implements ResponseInterface
         506 => 'Variant Also Negotiates',
         507 => 'Insufficient Storage',
         508 => 'Loop Detected',
+        510 => 'Not Extended',
         511 => 'Network Authentication Required',
+        599 => 'Network Connect Timeout Error',
```

Comment:
Doesn't appear to be "official"? Not sure what the stance is on this.
https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#Unofficial_codes